experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -103,7 +105,7 @@ function applySequenceFieldEdit(
 			const parent = navigateToNode(tree, change.parent);
 			assert(parent?.is(fuzzNode), "Defined down-path should point to a valid parent");
 			const field = parent.boxedSequenceChildren;
-			field.insertAt(change.index, [singleTextCursor(change.value) as any]);
+			field.insertAt(change.index, fieldCursorFromJsonableTrees([change.value]));
```

Comment:
Should `fieldCursorFromJsonableTrees` be exposed outside of the chunked-forest area? Seems like a generally useful function now, like `singleTextCursor`.

Reply:
yes at some point, but probably not that implementation which has some odd side effects on the path. I have added some todos about this to treeCursor utils.

Reply:
Sounds good, thanks.